packages/lesswrong/components/walledGarden/GatherTown.tsx

```diff
@@ -176,7 +180,9 @@ const GatherTown = ({classes}: {
           No users currently online. Check back later or be the first to join!
           {tooltip}
         </div>}
-        {!isMobile() && <GardenCodesList limit={2}/>}
```

Comment:
Looks like this line was rejected in review of https://github.com/LessWrong2/Lesswrong2/pull/3734 but merged in PR https://github.com/LessWrong2/Lesswrong2/pull/3769 which was based off of it. Anyways, this looks like the right fix.